articles/active-directory/connect/active-directory-aadconnectsync-feature-directory-extensions.md

```diff
@@ -41,10 +41,20 @@ An object in Azure AD can have up to 100 directory extensions attributes. The ma
 During installation of Azure AD Connect, an application is registered where these attributes are available. You can see this application in the Azure portal.  
 ![Schema Extension App](./media/active-directory-aadconnectsync-feature-directory-extensions/extension3new.png)
 
-These attributes are now available through Graph:  
+The attributes are prefixed with extension\_{AppClientId}\_. The AppClientId has the same value for all attributes in your Azure AD tenant.
+
+These attributes are now available through the **Azure AD Graph**:
+
+We can query the Azure AD Graph through the Azure AD Graph explorer: [https://graphexplorer.azurewebsites.net/](https://graphexplorer.azurewebsites.net/)
+
 ![Graph](./media/active-directory-aadconnectsync-feature-directory-extensions/extension4.png)
 
-The attributes are prefixed with extension\_{AppClientId}\_. The AppClientId has the same value for all attributes in your Azure AD tenant.
+Or through the **Microsoft Graph API**:
+
+We can query the Microsoft Graph API through the Microsoft Graph explorer: [https://developer.microsoft.com/en-us/graph/graph-explorer#](https://developer.microsoft.com/en-us/graph/graph-explorer#)
+
+>[!NOTE]
+> You need to explicitely ask for the attribute to be returned. This can be done by explicitly selecting the attributes like this: https://graph.microsoft.com/beta/users/abbie.spencer@fabrikamonline.com?$select=extension_9d98ed114c4840d298fad781915f27e4_employeeID,extension_9d98ed114c4840d298fad781915f27e4_division For more information please check [Microsoft Graph: Use query parameters](https://developer.microsoft.com/en-us/graph/docs/concepts/query_parameters#select-parameter)
```

Comment:
@tvuylsteke Unfortunately, you have another instance of "explicitly" misspelled in in the first sentence. Could you fix the typo.  I'll sign-off after the update.

Thanks,
Pat

Reply:
@PatAltimore Done. Terribly sorry! Thanks!